server/utils/utils.py

```diff
@@ -0,0 +1,65 @@
+"""General utility functions."""
+
+import asyncio
```

Comment:
@eliteprox I cleaned up the folder structure but this can also be done in a separate pull request.